README.md

```diff
@@ -264,15 +264,70 @@ Update coming...
+* **Use a validation set?** The regularization parameter can be tuned automatically. This requires a validation set to be defined using `DDLiteModel.set_holdout(validation_frac=p)` with `p > 0`, and either multiple or no values passed as the regularization parameter.
+
+ The rest of the parameters are as follows.
+
+### Common parameters to both joint and pipeline
+
+| **Parameter** | **Default value** | **Type** | **Notes** |
+|:---------|:--------|:---------|:--------|
+| `mu` | Set automatically if validation set,</br>or `1e-7` if no validation set | `float` or array-like, all non-negative | Sequence of or single regularization parameter</br>to use when training with features |
+| `bias` | `False` | `bool` | Include a bias term? |
+| `use_sparse` | `True` | `bool` | Use sparse matrix operations? |
+| `verbose` | `False` | `bool` | Print information during training? |
+| `plot` | `True` | `bool` | If using validation set to tune, show a diagnostic plot? |
+| `log` | `True` | `bool` | Log the learning results? |
+| `tol` | `1e-6` | `float` | Threshold of gradient-to-weights ratio to stop learning |
+|`sample` | `True` | `bool` | Use mini-batch SGD or full gradient? |
+|`n_samples` | `100` | `int` | Number of samples in mini-batch |
+|`alpha` | `0` | `float`, in `[0,1]` | Elastic-net mixing parameter (`0` is ridge, `1` is lasso) for feature training |
+|`warm_starts`| `False` | `bool`| Use warm starts if multiple `mu` values? |
+
+
+### Parameters for pipeline learning
+
+| **Parameter** | **Default value** | **Type** | **Notes** |
+|:---------|:--------|:---------|:--------|
+| `n_iter` | `500` | `int` | Default number of iterations for both stages |
+| `n_iter_lf` | `n_iter` | `int` | Number of iterations for LF stage |
+| `n_iter_feats` | `n_iter` | `int` | Number of iterations for feature stage |
+| `w0_mult_lf` | `1` | `float` | Multiplier for initial LF weights (`1` or `3` are typical) | 
+| `alpha_feats` | `alpha` | `float` | Same as `alpha` |
+| `rate` | `0.01` | `float` | Default learning rate for both stages |
+| `rate_lf` | `rate` | `float` | Learning rate for LF stage |
+| `rate_feats` | `rate` | `float` | Learning rate for feature stage |
+| `warm_starts_feats` | `warm_starts` | `bool` | Same as `warm_starts` |
+| `mu_lf` | `1e-7`, non-negative | `float` | Regularization parameter for LF stage |
+|`mu_seq_feats` | `mu` | `float` or array-like, all non-negative | Same as `mu`|
+| `n_mu_feats` | `5` | `int` | Number of `mu` values to fit if `mu` is `None` |
+|`mu_min_ratio_feats` | `1e-6` | `float` | Ratio of smallest to largest `mu` values to fit if `mu` is `None` |
```

Comment:
No need right now but probably a cleaner way to integrate all these duplicate joint/pipeline params...